cmd/xmidt-agent/default-config.yaml

```diff
@@ -0,0 +1,184 @@
+# SPDX-FileCopyrightText: 2023 Comcast Cable Communications Management, LLC
```

Comment:
formatting looks a little off (like spacing, i.e. `ping_write_timeout`), tried running a yaml linter? (if you haven't already 🙂 )

Reply:
do you know of one that doesn't remove comments?  I've tried a few and they all wipe out comments

Reply:
yeah... I just realized I also disabled mine because of that. I guess I got tired of it as well and manually formatted it 😭 

There has to be one that doesn't delete comments.

Reply:
there is some python one but it looked multi step and seemed like a pia

Reply:
I'll add a linter action to the repo for this file.